app/src/main/res/layout/activity_loading.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent">
```

Comment:
on high-resolution devices the gif looks small. any way to make the gif high bigger or change the aspect ratio?

Reply:
i will try this